playbooks/installation/pre-validation.adoc

```diff
@@ -0,0 +1,160 @@
+---
+---
+= OpenShift Pre-Install Validation Checklist
+Matt Witzenman <mwitzenm@redhat.com>
+v1.0, 2017-03-29
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+== Ensure ssh keys are propagated for ansible installer
+
+* The Ansible installer requires either:
+* root password-less ssh access using ssh keys 
+* A non-root user with password-less ssh acess and full password-less sudo access 
+
+----
+How to Propagate your key example:
+ssh-copy-id -i ~/.ssh/Myidrsa.pub remote.server.com
+----
+
+== DNS lookup on each node in your cluster
+
+* This is to ensure dns is properly set
+
+----
+example:
+dig www.google.com +short
+172.217.7.132
+----
+
+== DNS reverse lookup on each node 
+
+----
+example:
+dig -x `dig www.google.com +short` +short
+iad23s59-in-f4.1e100.net.
+----
+
+== OCP 3.3 install repos
+
+* If Installing OpenShift 3.3 please use these repos
+
+----
+subscription-manager repos \
+    --enable="rhel-7-server-rpms" \
+    --enable="rhel-7-server-extras-rpms" \
+    --enable="rhel-7-server-optional-rpms"\
+    --enable="rhel-7-server-ose-3.3-rpms"
+----
+
+== OCP 3.4 install repos
+
+* If Installing OpenShift 3.4 please use these repos
+
+----
+subscription-manager repos \
+    --enable="rhel-7-server-rpms" \
+    --enable="rhel-7-server-extras-rpms" \
+    --enable="rhel-7-server-optional-rpms"\
+    --enable="rhel-7-server-ose-3.4-rpms"
+----
+
+== Show the repos expected and enabled
+
+----
+yum repolist enabled
+----
+
+== Are updates required before you install OpenShift
+
+----
+yum list updates
+----
+
+== Is subscription manager active
+
+----
+subscription-manager status
+----
+
+== To list all repos recognized by your rhel 7 nodes
+
+----
+subscription-manager repos
+----
+
+== Install some extras required for OpenShift
```

Comment:
I am confused ...I did not realize the installer actually installs everything it needs?